flight/Libraries/math/lqg.c

```diff
@@ -0,0 +1,515 @@
+/**
+ ******************************************************************************
+ * @addtogroup Libraries Libraries
+ * @{
+ * @addtogroup FlightMath math support libraries
+ * @{
```

Comment:
These need to be closed at the end of file.

Reply:
Now I know what those weird comments at the end of files are even good for.